tests/unit/stream_alert_shared/test_rule.py

```diff
@@ -0,0 +1,225 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+# pylint: disable=no-self-use,protected-access
+from mock import patch
```

Comment:
`mock` in Python 2 is a third-party library, it should be imported in the same group as `nose.tools`. So the import order would be:

```python
from mock import patch
from nose.tools import assert_equal, raises

from stream_alert.shared import rule
```
I'm surprised pylint didn't complain

Reply:
(this happens in other test files as well)

Reply:
good call - I'll address all of them :)